src/lifecycles/update.js

```diff
@@ -0,0 +1,28 @@
+import { UPDATING, MOUNTED, SKIP_BECAUSE_BROKEN } from '../applications/app.helpers.js';
+import { handleAppError, transformErr } from '../applications/app-errors.js';
+import { reasonableTime } from '../applications/timeouts.js';
+import { getProps } from './prop.helpers.js';
+
+export function toUpdatePromise(appOrParcel) {
+  return Promise.resolve().then(() => {
+    const objectType = appOrParcel.isParcel ? 'parcel' : 'application';
```

Comment:
I think we may want to throw an error here if they appOrParcel is an application, assuming we're only supporting the update lifecycle on a parcel. 

Do you see any reason we should support update on an application?

Reply:
No I don't see any reason for this to be called on an application. I was following the pattern from other files, but there's no need to here.

I think instead of throwing an error it would make more sense just to never call update on an application. The error would only be thrown if there is a bug in the single-spa code itself, since this function is not exposed externally. I like being defensive when it's external and something other code could use, but when it's internal usage I prefer not implementing all the defensive checks.

I'll change my code so it doesn't do this check and i'll rename the variable to just `parcel` instead of application.